pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -210,3 +223,18 @@ func newCDIForCR(cr *hcov1alpha1.HyperConverged) *cdi.CDI {
 		},
 	}
 }
+
+// Verify if the value is a valid PullPolicy
+func verifyPullPolicyType(policy string) (bool, error) {
```

Comment:
why do you return the bool if you never use it?

Reply:
Simply because I prefer to use a regular convention, returning error as well.